core/src/main/java/gyro/core/validation/CollectionMin.java

```diff
@@ -0,0 +1,15 @@
+package gyro.core.validation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@ValidatorClass(CollectionMinValidator.class)
+public @interface CollectionMin {
```

Comment:
Let's use the full form instead of abbreviation, so rename this `CollectionMinimum`.

Reply:
Never mind. we already have `Min` and `Max` annotations.